korangar/src/input/mod.rs

```diff
@@ -31,6 +31,7 @@ use crate::interface::resource::PartialMove;
 
 const MOUSE_SCOLL_MULTIPLIER: f32 = 30.0;
 const KEY_COUNT: usize = variant_count::<KeyCode>();
+const DOUBLE_CLICK_TIME_MS: u32 = 300;
```

Comment:
Can be config-ed later?

Reply:
The default doube click time-frame on windows is 500 ms. Since most poeple most likely have this in their muscle memory, we should leave it as 500 ms too.

[Source](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setdoubleclicktime)

Reply:
google ai failed me